tests/integration/fork/conftest.py

```diff
@@ -11,7 +11,7 @@
 @pytest.fixture(scope="module", autouse=True)
 def forked_env():
     with boa.swap_env(Env()):
-        fork_uri = os.environ.get("MAINNET_ENDPOINT", "http://localhost:8545")
-        blkid = 17467922  # some block we know the state of
+        fork_uri = os.environ.get("MAINNET_ENDPOINT") or "http://localhost:8545"
```

Comment:
why does this need to be changed?

Reply:
The env var is always given because of this line: https://github.com/vyperlang/titanoboa/blob/18007cd8ff5e5954907a7725f1c18b51da042e20/.github/workflows/main.yaml#L59

However, when it's empty you get some error about the URL being invalid, instead of properly using the fallback URL